spec/models/provider_spec.rb

```diff
@@ -164,7 +165,8 @@
       end
     end
 
-    context "when passed an existing user" do
+    # looks like  SILAS will pass a single office code as single element array not a string, but just in case...
```

Comment:
If this comment was based on my payload file... I may have updated it to test a theory and forgotten to revert it! 😬